src/sage/groups/group.pyx

```diff
@@ -42,8 +42,7 @@ def is_Group(x):
         sage: is_Group("a string")
         False
     """
-    from sage.groups.old import Group as OldGroup
-    return isinstance(x, (Group, OldGroup))
+    return isinstance(x, Group)
```

Comment:
Is this correct?

Shouldn't we rather keep it and deprecate `is_Group` (see https://github.com/sagemath/sage/issues/32414)?

Reply:
I will deprecate the version with only `Group`.